cmd/kube-scheduler/app/options/options.go

```diff
@@ -217,120 +205,17 @@ func (o *Options) Validate() []error {
 	return errs
 }
 
-// Config return a scheduler config object
-func (o *Options) Config() (*schedulerappconfig.Config, error) {
+// Initialize initializes Option's scheduler app configuration.
+func (o *Options) Initialize() error {
```

Comment:
Maybe we could add a comment that says this must be called after flags have been parsed?

Reply:
Done